src/main/java/com/quizapp/enterprise/controllers/GameViewController.java

```diff
@@ -18,7 +18,7 @@
     @RequestMapping("gameView/{gameId}")
     public String game(@PathVariable("gameId") String gameId, Model model){
 
-        model.addAttribute("players",gameService.getGame(gameId).getPlayers());
+        model.addAttribute("players", gameService.getGame(gameId).getPlayers());
```

Comment:
added space after argument within parameters for best practices and readability.
This is recommended by the "Spacing & line breaks" section of the java best practices summary referenced below.

source: https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3